src/helpers/getIconType.js

```diff
@@ -17,8 +16,6 @@ export default (type) => {
       return OcticonIcon
     case 'material':
       return MaterialIcon
-    case 'material-community':
```

Comment:
@jenessawhite What's the reason for this change? If this is unrelated please revert it.